guides/source/routing.md

```diff
@@ -592,7 +592,7 @@ TIP: By default, dynamic segments don't accept dots - this is because the dot is
 
 ### Static Segments
 
-You can specify static segments when creating a route by not prepending a colon to a segment:
+You can specify static segments when creating a route by not prepending a colon to a fragment:
```

Comment:
It looks like the changes in this file got picked up from #39775?

Reply:
Ah thanks 👍 . I've removed these changes.